### DISCUSSION TITLE
我的设备是iStoreOS，yml和env都改好了，但是docker里没有项目。如何继续呢？

### DISCUSSION THREAD

**Discussion by maodi0614** [monlor/docker-xiaoya#209] [Q&A]
我的设备是iStoreOS，yml和env都改好了，但是docker里没有项目。如何继续呢？
群晖的里面有项目，可以新建。但是istoreos没有，请问如何操作？

**Comment by xmh0754** [comment]
> 我的设备是iStoreOS，yml和env都改好了，但是docker里没有项目。如何继续呢？ 群晖的里面有项目，可以新建。但是istoreos没有，请问如何操作？
我的也是iStoreOS，使用一键部署脚本就可以了，很方便。
